test-suite-groovy/src/test/groovy/io/micronaut/docs/netty/LogbookPipelineCustomizer.groovy

```diff
@@ -30,6 +29,7 @@ class LogbookPipelineCustomizer
         ChannelPipelineCustomizer customizer = event.bean
 
         if (customizer.serverChannel) { // <2>
+            /*
             customizer.doOnConnect( { ChannelPipeline pipeline ->
                 pipeline.addAfter(
                         ChannelPipelineCustomizer.HANDLER_HTTP_SERVER_CODEC,
```

Comment:
Do we want to leave these commented sections? I assume they'll appear in the docs?

Reply:
🤷